src/runtime/manifest-parser.peg

```diff
@@ -73,14 +73,14 @@
   }
 
   function requireUsePreSlandleSyntax() {
-    if (!Flags.usePreSlandlesSyntax) {
-      expected('pre slandles syntax');
-    }
+    /*if (!Flags.usePreSlandlesSyntax) {
+      error('using pre slandles syntax (disabled by flag)');
+    }*/
```

Comment:
I've been toggling this on and off during testing so that when we 'flip the switch' our tests keep working. If I could keep this little bit of tech debt around for the week or two I'd appreciate it.